src/shared/scrapers/US/NYT.js

```diff
@@ -25,26 +25,25 @@ const scraper = {
     const data = await fetch.csv(this.url, false);
 
     // FIXME when we roll out new TZ support!
```

Comment:
Does this resolve the fixme or not yet?

Reply:
I have no idea what is the fixme about, TZ is a big project of @ryanblock.

Reply:
It does not, but that is in flight!